apps/course-authoring/types.d.ts

```diff
@@ -1 +1,6 @@
-declare module '*.scss'
\ No newline at end of file
+declare module '*.scss'
```

Comment:
I think this `types.d.ts` is "too global", we could move it to the same folder as the `index.ts` file `/src/main` and call it `index.types.ts` for consistency.

Also, as we'll be getting rid of the `exportScormApi` and replacing it with a new method in the `window.electron.ipcrender` we can create something more generic for the type, kinda like:
```ts
declare global {
  interface Window {
    electron: {
      ipcRenderer: {
        [key: string]: (...args: unknown[]) => void;
      };
    };
  }
}

export {};
```

You'll notice TS angry at you when importing styles, I'm pretty sure it's been fixed in the new owlUI version that will be released soon.